src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private int firstSide;
+    private int secondSide;
+
+    public Rectangle(int firstSide, int secondSide, Color color) {
+        this.firstSide = firstSide;
+        this.secondSide = secondSide;
+        super.color = color;
+    }

```

Comment:
It's a good practice to add a getter and setter for each property in a class to access and modify them. Consider adding getters and setters for 'firstSide' and 'secondSide'.

Reply:
I had them, but AI asked me to remove them if they were not used.